tests/endpoint-test.js

```diff
@@ -283,6 +283,45 @@ describe('Endpoint', function() {
 				});
 			});
 
+			it('should make correct request with additional parameters and correctly parse response when API call is successful', function(done) {
```

Comment:
Endpoint tests are generally expensive, so we only do one happy path test per endpoint.  Tests like these should be unit tests instead